src/components/Dropdown/DropdownMenu.react.js

```diff
@@ -52,14 +53,27 @@ function DropdownMenu({
     className
   );
   return (
-    <div
-      className={classes}
-      data-placement={position}
-      style={style}
-      ref={rootRef}
-    >
-      {children}
-    </div>
+    <Popper placement={position} eventsEnabled={true} positionFixed={false}>
+      {({
+        ref,
+        style,
+        placement,
+        arrowProps,
```

Comment:
[nit] this looks like an unused variable - if so we can just remove it for readability